src/components/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { StyledButton } from './styles/Button.styled'
+
+type Props = {
+  filled: boolean
+  buttonText: string
+}
+
+const Button = (props: Props) => {
```

Comment:
Consider naming it "FilledButton", which seems like the purpose of your code

Reply:
Fixed in commit b6c83a4